engine/engine.go

```diff
@@ -161,11 +165,23 @@ func (e *Engine) Start(ctx context.Context) error {
 
 	e.cache = dsCache
 
-	e.publisher, err = dtsync.NewPublisherFromExisting(e.dataTransfer, e.host, e.pubSubTopic, e.lsys)
+	if e.httpPublisherCfg.Enabled {
```

Comment:
So means Provider would start either a HTTP publisher or `dtsync.Publusher`? 

I think we need to document this.

Reply:
Changed in 0a8a8bb1697d77109f30f20627fc034e77da98f4 with an enum field in the ingest cfg.